common.gypi

```diff
@@ -38,7 +38,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.17',
+    'v8_embedder_string': '-node.21',
```

Comment:
This is to correct the number after https://github.com/nodejs/node/commit/573171deb047144282c30a67976cfad2b701e48d was cherry-picked incorrectly.